index.html

```diff
@@ -4,9 +4,58 @@
+            </div> 
+        <span class="orange-theme-color nav-text brand-name">dreamUI</span>
+        </div>
+        <div class="nav-bar-tabs mr-3">
+            <p class="nav-text nav-tab-text">Home</p>
+            <p class="nav-text nav-tab-text">Documentation</p>
+            <a href="https://github.com/dikshadhote/dreamUI/tree/dev"><i class="fab fa-github nav-text nav-tab-text" title="GitHub"></i></a>
+        </div>
+
+    </nav>
+    <div class="hero-img-container">
```

Comment:
A suggestion, since this is the main section of your HTML, you can use `<main></main>` tags
Would be better, semantic wise

Reply:
Ohh, thanks, I didn't thought this. Incorporated this suggestion